hedera-node/hedera-mono-service/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -362,12 +362,8 @@ private void internalInit(
                 // Once we have a dynamic address book, this will run unconditionally
                 app.sysFilesManager().updateStakeDetails();
             }
-            if (trigger == RESTART && isUpgrade) {
```

Comment:
It's moved to [here](https://github.com/hashgraph/hedera-services/blob/virtualize-more-maps/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/ServicesState.java#L226), because if `accounts.storeOnDisk=true` and the `BackedSystemAccountsCreator` tries to create an account e.g. [here](https://github.com/hashgraph/hedera-services/blob/virtualize-more-maps/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/initialization/BackedSystemAccountsCreator.java#L100) **before** we have migrated to `VirtualMap`, we'll get a `ClassCastException`.